domain/servicefactory/controller.go

```diff
@@ -196,3 +198,10 @@ func (s *ControllerFactory) Macaroon() *macaroonservice.Service {
 		clock.WallClock,
 	)
 }
+
+// Agent returns the model agent service.
```

Comment:
is model agent service correct ?

Reply:
This is a question of terminology. This function returns the `*modelagentservice.Service` which is **not** scoped to a specific model, and so it's able to retrieve the agent version for **any** model. I thought we were still calling this "model agent service", considering the package is called `modelagent`. But what do you think would be a clearer name/comment here?